### DISCUSSION TITLE
Add GitHub Actions SetOutput Command

### DISCUSSION THREAD

**Discussion by gitfool** [cake-build/cake#3984] [Ideas]
Feature request: create or update an output for any steps running next in a job to avoid the following warning with latest runner:
```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files.
```

References:
* [GitHub Actions: Deprecating save-state and set-output commands](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands)
* [Workflow commands for GitHub Actions: Setting an output parameter](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter)

**Comment by devlead** [comment]
Are we talking about the Cake Common GitHubActions Build provider? It currently uses an environment file for the SetEnvironmentVariable command

https://github.com/cake-build/cake/blob/50dc5604337971656da45206c050032718298893/src/Cake.Common/Build/GitHubActions/Commands/GitHubActionsCommands.cs#L80-L104

And similar should be done for [Setting an output parameter](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) using `GITHUB_OUTPUT` variable?

**Reply by gitfool** [reply]
Exactly.

**Reply by devlead** [reply]
Sounds like a reasonable addition to me, something you would like to send a PR for? If so please raise an issue and I'll assign it to you.
